src/src/facilitator/facilitator.service.ts

```diff
@@ -0,0 +1,255 @@
+import { HttpService } from '@nestjs/axios';
+import { Injectable } from '@nestjs/common';
+import { lastValueFrom, map } from 'rxjs';
+import { EnumService } from '../enum/enum.service';
+import { HasuraService } from '../hasura/hasura.service';
+
+@Injectable()
+export class FacilitatorService {
+  public url = process.env.HASURA_BASE_URL;
```

Comment:
use config service like: this.configService.get<string>('HASURA_BASE_URL');

Reply:
Done